ExploringSettings/Settings_TestExploringSettings.m

```diff
@@ -0,0 +1,26 @@
+classdef Settings_TestExploringSettings < SettingsNumericalHomogenizer
+    
+    %  ____                      
+    % / ___|_      ____ _ _ __   
+    % \___ \ \ /\ / / _` | '_ \  
+    %  ___) \ V  V / (_| | | | | 
+    % |____/ \_/\_/ \__,_|_| |_|
+    %               Topology Optimization Laboratory
+    %
+    % Specific implementation of NumHomog settings for a specific case
+    % Here we set the specific parameters that configure our test
+    % This kind of files would replace our current Benchmark files
+    
+    properties
+    end
+    methods
+        function obj = Settings_TestExploringSettings()
+            obj.outFileName = 'test2d_micro';
```

Comment:
This file would be the actual implementation of the parameters for an specific benchmark or test. These could also be read from a file (as discussed in our previous meeting).